maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -58,7 +58,8 @@
 @Mojo(name = "incrementalify", requiresDirectInvocation = true, aggregator = true)
 public class IncrementalifyMojo extends AbstractVersionsUpdaterMojo {
 
-    private static final String MINIMUM_PARENT = "3.10";
+    private static final String MINIMUM_PLUGIN_PARENT = "3.10";
+    private static final String MINIMUM_JENKINS_PARENT = "1.47";
```

Comment:
1.43 really, as of https://github.com/jenkinsci/pom/pull/20. [Changelog](https://github.com/jenkinsci/pom/blob/master/CHANGELOG.md) 1.47 does not yet exist. Are there some merged PRs you think must be released in order for it to work at all?

Reply:
reincrementalify mojo 👍

Reply:
I suppose you mean that you want this merged and released and then will file a `pom` PR to configure this Maven plugin version?

Note that `mvn incrementals:reincrementalify` is just a convenience. You can do the same thing by hand, or (in a large multimodule reactor) use `versions:set`.

Reply:
Yes, indeed 😄

Reply:
Sorry, `incrementalify` is what I meant—the one-off goal that you are patching here.

Reply:
Yes I could run the one-off goal indeed just by using the SNAPSHOT locally 👍